odoo/tools/translate.py

```diff
@@ -888,6 +891,72 @@ def handle_text(text, lineno):
     _extract_translatable_qweb_terms(tree.getroot(), handle_text)
     return result
 
+
+def extract_formula_terms(formula):
+    """Extract strings in a spreadsheet formula which are arguments to '_t' functions
+    >>> extract_formula_terms('=_t("Hello") + _t("Raoul")')
+    ["Hello", "Raoul"]
+    """
+    terms = []
+    tokens = tokenize(io.BytesIO(formula.encode('utf-8')).readline)
```

Comment:
Use `generate_tokens`, it takes a string reader. also doesn't have the ENCODING token we don't care about.